distribution/build.py

```diff
@@ -123,10 +123,6 @@ def get_dependencies():
         # TODO: Remove this once we have a stable PyPI index for trustyai_fms
         # Fix trustyai_fms package to use test PyPI index
```

Comment:
Can you remove this as well please?

Reply:
good catch, sorry I am jumping around many things

Reply:
fixed 👍🏽